guide/blueprints/catalog/index.md

```diff
@@ -375,153 +375,120 @@ the latest non-snapshot version will be loaded when an entity is instantiated.
 
 ### Bundling Catalog Resources
 
-When deploying a blueprint, it is possible to deploy it as an OSGI bundle with additional resources scoped
-to this bundle. This is particularly useful when, for example, deploying a Tomcat server with a `.war` file
-which is deployed using a classpath path such as `classpath://mywar.war`.
+It is possible to add an OSGi bundle to AMP. This is useful when you have a blueprint that needs to reference external scripts/resources or when you have multiple blueprints that you want to keep in sync. Brooklyn will persist any uploaded bundles so that they are available after a restart, or a HA failover.
 
-In this example, we will create a simple `server` catalog item, bundled with a simple text file.
+In this example, we will create a simple `my-server` catalog item, bundled with a simple script. The script will be run when launching the server.
 
-First, create the text file with some sample contents:
+First, create a folder called bundleFolder, then add a file called myfile.sh to it. 
+The contents of myfile.sh should be as follows:
 
 ~~~ bash
-echo Hello, World! > myfile.txt
+echo Hello, World!
 ~~~
 
-Now create a file called `catalog.bom` with the following contents:
+Now create a file in bundleFolder called `catalog.bom` with the following contents:
 
 ~~~ yaml
 brooklyn.catalog:
-  bundle: script-server
+  bundle: MyServerBundle
   version: 1.0.0
-  items:
-  - id: script-server
-    itemType: entity
-    item:
-      type: server
-      brooklyn.config:
-        files.runtime:
-          classpath://myfile.txt: files/myfile.txt
+  item:  
```

Comment:
The [catalog docs](http://brooklyn.apache.org/v/latest/ops/catalog/index.html) use the following format rather than the shortcut format which excludes `items:`

```
items:
  id: foo
  item:
    type: bar
```